src/System.Diagnostics.Process/src/System/Diagnostics/ProcessThread.OSX.cs

```diff
@@ -32,8 +43,11 @@ public TimeSpan PrivilegedProcessorTime
         {
             get
             {
-                // TODO: Implement this
-                throw NotImplemented.ByDesign;
+                Interop.libproc.proc_threadinfo? info = Interop.libproc.GetThreadInfoById((ulong)_threadInfo._threadId);
+                if (info.HasValue)
+                    return new TimeSpan((long)info.Value.pth_system_time);
```

Comment:
Yes, these values seem to be Ticks as well (and the return values, converted to a TimeSpan) look to be correct values. I say "seem" and "look" because there doesn't seem to be documentation about the specific struct values in the header file.